src/Microsoft.ML.Tokenizers/Directory.Build.props

```diff
@@ -7,8 +7,7 @@
 
   <PropertyGroup>
     <MajorVersion>$([MSBuild]::Subtract($(MajorVersion), 3))</MajorVersion>
-    <!-- Blank version for now for API compat -->
-    <PackageValidationBaselineVersion></PackageValidationBaselineVersion>
+    <PackageValidationBaselineVersion>$(MajorVersion).$(MinorVersion).$([MSBuild]::Subtract($(PatchVersion), 1))</PackageValidationBaselineVersion>
```

Comment:
>$([MSBuild]::Subtract($(PatchVersion), 1) [](http://example.com/codeflow?start=70&length=41)

This looks ok but I am wondering if we can always use `0` for the patch version here?

Reply:
I specifically chose last version so that we're less likely to be in a situation where we are restoring an out of support version that will be flagged by compliance tooling.